tests/test_transforms.py

```diff
@@ -66,6 +66,34 @@ def test_mask_types(mask_type):
             assert num_low_frequencies == expected_num_low_frequencies
 
 
+@pytest.mark.parametrize(
+    (
+        "sync_fractions_accel, center_fractions, accelerations, seed, "
+        "choose_acceleration_output"
+    ),
+    [
+        (False, [0.04, 0.08], [8, 4], 2, (0.04, 4)),
+        (True, [0.04, 0.08], [8, 4], 2, (0.04, 8)),
```

Comment:
For the true case, one approach would be to loop a bit and assert and no-cross terms appear, but that may be too much.